docs/changelog/113218.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 113218
+summary: Forbid - in namespace in reroute processor
+area: Ingest Node
+type: bug
```

Comment:
Not sure if "bug" is the best description. This is going to break use cases where people are using non-compliant namespace with `-` in it.

Reply:
@mattc58 should it be something else other than "bug"?